app.py

```diff
@@ -68,12 +72,89 @@
     _LOGGER.info("installation=%s", RUNTIME_CONTEXT.app_installation)
 
 
+@process_event_actions("pull_request", {"opened", "reopened", "synchronize", "edited"})
+@process_webhook_payload
+async def on_pr_open_or_sync(*, action, number, pull_request, repository, sender, organization, installation, **kwargs):
```

Comment:
Where does the `organization` come from? It is not defined https://developer.github.com/v3/activity/events/types/#pullrequestevent and it looks like it's not sent along with the event either:
```
Traceback (most recent call last):
  File "/home/macermak/.local/share/virtualenvs/Qeb-Hwt-mdTlIz7D/lib/python3.7/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "/home/macermak/.local/share/virtualenvs/Qeb-Hwt-mdTlIz7D/lib/python3.7/site-packages/octomachinery/app/routing/webhooks_dispatcher.py", line 82, in wrapper
    return await wrapped_function(request, github_app=github_app)
  File "/home/macermak/.local/share/virtualenvs/Qeb-Hwt-mdTlIz7D/lib/python3.7/site-packages/octomachinery/app/routing/webhooks_dispatcher.py", line 120, in route_github_webhook_event
    await dispatch_event(event)
  File "/home/macermak/.local/share/virtualenvs/Qeb-Hwt-mdTlIz7D/lib/python3.7/site-packages/gidgethub/routing.py", line 80, in dispatch
    await callback(event, *args, **kwargs)
  File "/home/macermak/.local/share/virtualenvs/Qeb-Hwt-mdTlIz7D/lib/python3.7/site-packages/octomachinery/app/routing/__init__.py", line 27, in wrapper
    return original_function(*args, **kwargs)
  File "/home/macermak/.local/share/virtualenvs/Qeb-Hwt-mdTlIz7D/lib/python3.7/site-packages/octomachinery/app/routing/decorators.py", line 10, in wrapper
    return wrapped_function(**event.data)
TypeError: on_pr_open_or_sync() missing 1 required keyword-only argument: 'organization'
```

Reply:
@CermakM organization only appears in webhook payloads sent to GitHub Apps because they are installed into orgs or user accs. It's probably best to use `**kwargs` to consume unused args. This is not really documented in the event types because this would be missing in webhooks sent to other receivers.
I think this is only mentioned here: https://developer.github.com/webhooks/#payloads.

Reply:
@webknjaz Thanks for the explanation! Indeed, I haven't received the `organization` as part of my payload (as I configured it for a single repository).